deploy/local/live-csc/docker-compose.yml

```diff
@@ -111,7 +111,15 @@ services:
       - mtcs-sim
     environment:
       <<: *base-producer-environment
-      LOVE_CSC_PRODUCER: MTHexapod:2 
+      LOVE_CSC_PRODUCER: MTHexapod:2
+  MTDome:
+    <<: *base-producer
+    container_name: producer-mtdome-image-mount
+    depends_on:
+      - mtcs-sim
+    environment:
+      <<: *base-producer-environment
+      LOVE_CSC_PRODUCER: MTDome:1
```

Comment:
This should be `MTDome:0` as it only exist 1 instance of the Main Telescope Dome